site/layouts/shortcodes/svg-docs.html

```diff
@@ -1,48 +1,24 @@
 {{- /*
-  Usage: `svg-docs file="img/orange-logo.svg" downloadable="true"`
+  Usage: `svg-docs file="site/static/docs/{version}/assets/brand/orange-logo" downloadable="true"`
 
     Prints SVG content.
 
     Optional parameters:
-      file: SVG file path - default: `img/orange-logo.svg`
+      file: SVG file path without file extension - default: `site/static/docs/{version}/assets/brand/orange-logo`
       downloadable: is the file downloadable? - default: `true`
 */ -}}
 
-{{- $file := .Get "file"| default "img/orange-logo.svg" -}}
-{{- $svg := readFile $file -}}
-{{- $code := $svg -}}
+{{- $file := .Get "file" | default (print "site/static/docs/" .Site.Params.docs_version "/assets/brand/orange-logo") -}}
+{{- $svgFormatted := readFile (print $file "-formatted.svg") -}}
+{{- $svg := readFile (print $file ".svg") -}}
+{{- $code := $svgFormatted -}}
```

Comment:
Couldn't we call `$svgFormatted`, `$code` directly ?

Reply:
Removed entirely `$code` to keep `$svg` and `$svgFormatted` that help in terms of readability in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1065/commits/a61d6708730cb2f4420ac976e479efb9a02f8517.